deployer/commands/generate/resource_allocation/node-capacity-info.json

```diff
@@ -70,5 +70,41 @@
             "cpu": 3.45,
             "memory": 28967406928
         }
+    },
+    "n2-highmem-16": {
+        "capacity": {
+            "cpu": 16.0,
+            "memory": 135081324544
+        },
+        "allocatable": {
+            "cpu": 15.89,
+            "memory": 125139775488
+        },
+        "measured_overhead": {
+            "cpu": 0.447,
```

Comment:
The current GKE overhead per node is 454m and 656Mi i think, i expect issues using this atm :/

Reply:
@consideRatio ah, I see. I would like us to not get too much into this, and will suggest a different method forward for now. I think working on this autogenerated part should be treated as a product effort and prioritized accordingly, and I'll bring that up with Giuliano tomorrow as well.

Reply:
Suggested https://github.com/2i2c-org/infrastructure/pull/4131#discussion_r1616621073

Reply:
Thanks @yuvipanda I'll drop thinking about this!